app/api/tracks.py

```diff
@@ -64,13 +66,24 @@ def valid_color(self, data):
                             type_='session')
 
 
-class TrackList(ResourceList):
+class TrackListPost(ResourceList):
     """
     List and create Tracks
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
```

Comment:
@shubham-padia Shouldn't this be `track co-oragnizer` ? Not sure, correct me if I am wrong.